index.html

```diff
@@ -0,0 +1,99 @@
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./style/main.css" />
+  <title>S. Peterson School of Modern Art Exhibition</title>
+</head>
+<body>
+  <div class="menu-mobile">
+    <div>
+      <a href="#"><img  class="burger-image" src="./image/mobile_menu.png"></a>
+    </div>
+    <div>
+      <a href="#"><img class="top-logo" src="./image/logo-small.png" alt="SPS MA Exhibition 2023"></a>
+    </div>
+  </div>
+
+  <nav class="mobile-menu-box">
+    <a href="#"><img  class="close-image" src="./image/close_mobile_menu.png"></a>
+    <ul class="nobull">
+      <li class="mobile-menu-item"><a class="mobile-menu-item-text" href="#">Home</a></li>
+      <li class="mobile-menu-item"><a class="mobile-menu-item-text" href="./about.html">About</a></li>
+      <li class="mobile-menu-item"><a class="mobile-menu-item-text" href="#program">Program</a></li>
+      <li class="mobile-menu-item"><a class="mobile-menu-item-text" href="#speakers">Speakers</a></li>
+      <li class="mobile-menu-item"><a class="mobile-menu-item-text" href="#partners">Partners</a></li>
+    </ul>
+  </nav>
+
+  <div class="menu-desktop">
+    <div>
+      <a href="#"><img class="top-logo" src="./image/logo-small.png" alt="SPS MA Exhibition 2023"></a>
+    </div>
+
+    <nav class="desktop-menu-box">
+      <ul class="nobull">
+        <li class="desktop-menu-item"><a class="desktop-menu-item-text" href="#">Home</a></li>
+        <li class="desktop-menu-item"><a class="desktop-menu-item-text" href="./about.html">About</a></li>
+        <li class="desktop-menu-item"><a class="desktop-menu-item-text" href="#program">Program</a></li>
+        <li class="desktop-menu-item"><a class="desktop-menu-item-text" href="#speakers">Speakers</a></li>
+        <li class="desktop-menu-item"><a class="desktop-menu-item-text" href="#partners">Partners</a></li>
+      </ul>
+    </nav>
+  </div>
+
+  <section class="invite">
+    <h3>"Hello, Art World!"</h3>
+    <h1>S. PETERSON SCHOOL <br>OF MODERN ART <br>EXHIBITION 2023</h1>
+    <p class="invite-text">An exhibition of contemporary art works where people 
+      from around the globe who believe in the value of art 
+      and creativity gather and share own masterpieces, opinions, ideas and feedback.
+    </p>
+    <p class="invite-date">Since 2022.10.31 open 7/24</p>
+    <p class="invite-location">@ World Wide Web Site of Serg Skol
+    </p>
+  </section>
+
+  <section class="program" id="program">
+    <h2>Main Program</h2>
+    <div class="guide-bar"></div>
+    <div class="program-list"></div>
+    <div class="program-view-full">View full program ></div>
+  </section>
```

Comment:
- [ ] `required`
- [ ] Kindly the boxes fit as the original template by giving them even space around
- [ ] Also change the word `view programs` to `see the whole program` and it should be underlined and at the center
- [ ] please see screenshots below

`your template`
![image](https://user-images.githubusercontent.com/66230306/198033993-86a68ea8-a127-4b7c-9580-b016648b242b.png)

`original template`
![image](https://user-images.githubusercontent.com/66230306/198034890-5bc14a3c-3d00-4bf6-b3f3-8d802399a637.png)

Reply:
Thanks! It's done.